pkg/controllers/networkpolicy/sync.go

```diff
@@ -130,15 +130,21 @@ func (a *appliedToSecurityGroup) sync(syncContent *securitygroup.Synchronization
 
 	// roughly count and compare rules in syncContent against nps.
```

Comment:
whats with roughly.. Isnt it accurate calculation?

Reply:
It's not accurate calculation. It compares based on item counts for all rules, not per rule and has no context on network policies.